docs/conductor/getting-started.md

```diff
@@ -0,0 +1,93 @@
+# Getting started with Conductor
+##
+
+Conductor will group and schedule automated Dependency Updates on your own Continuous Integration platform. If the update succeeds, Conductor sends you a pull request to your code hosting platform (GitHub, GitLab, Bitbucket) with the changed composer.lock file and, if necessary, other files modified by Composer plugins or scripts.
+
+To use Conductor:
+
+- You need to be approved for early access to conductor. [Join to the waitlist](http://packagist.com.lo/features/conductor) and wait for approval.
+- You need a Private Packagist trial or subscription (Cloud or Self-Hosted).
+- You need to add a workflow to your Continuous Integration platform to run Composer updates, described below.
+
+## First steps
+
+Once you have a Private Packagist Subscription and Conductor is enabled for your subscription, log into your Private Packagist organization and click on the "Updates" tab in the main navigation.  
+Conductor will list packages:
+ - added to Private Packagist via synchronization
+ - having a composer.lock file commited to the repository
+ - having the Private Packagist repository added to the composer.json
```

Comment:
```suggestion
Conductor will manage dependencies for packages added to Private Packagist via synchronization that have a composer.lock file committed to the repository.
```

Looking at the list it isn't clear if a package needs to match one or all of the criteria. As we are already working on the third point, I would drop that already.